example/project.godot

```diff
@@ -13,6 +13,7 @@ config_version=5
 config/name="TiltFiveGodot4"
 run/main_scene="res://main.tscn"
 config/features=PackedStringArray("4.1")
+run/max_fps=60
```

Comment:
Did you mean to add this in?

Reply:
Not really but it shouldn't do any harm. I've been told the T5 won't run over 60fps.